.github/workflows/create_realease.yml

```diff
@@ -133,7 +159,7 @@ jobs:
           tag: ${{ inputs.tag || needs.get_context_info.outputs.DATE_REV }}-LIVE
           commit: ${{ needs.get_context_info.outputs.COMMIT_LIVE }}
           name: ${{ needs.get_context_info.outputs.RELEASE_TITLE_LIVE }}
-          body: ${{ vars.RELEASE_BODY_LIVE }}
+          body: ${{ vars.RELEASE_BODY_LIVE }} [Änderungen zum letzten Release](${{ needs.get_context_info.outputs.LINK_LIVE }})
```

Comment:
Ich würde das @rjcncpt reinschreiben lassen, damit hat er dann in der neuen Zeile direkt den Text (und man kann später ohne commit noch das Wording anpassen)

Reply:
Genau, solange der MD Link noch ordentlich angezeigt wird (sollte aber :-D)